apps/remix-ide/src/app/tabs/analysis-tab.js

```diff
@@ -28,6 +28,9 @@ class AnalysisTab extends ViewPlugin {
   }
 
   render () {
+    const listOfPlugins = localStorage.getItem('workspace')
```

Comment:
is it the only way to do it? accessing the localstorage directly here sounds risky

Reply:
and the name `solidityStaticAnalysis` is not really tied to the view itself